german/Korallen/Korallen.md

```diff
@@ -0,0 +1,7 @@
+Sie tauchen nochmal, um die schönen Korallen näher zu betrachten. Dabei entdecken Sie eine Höhle unter der Wasseroberfläche, in der Sie einen goldenen Schimmer auszumachen glauben.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.